### ISSUE TITLE
[6.x] Crash at startup

### ISSUE THREAD

**Issue by msoky** [TrinityCore/TrinityCore#15072]
i have crash at startup, can anybody help?
![vystrizek](https://cloud.githubusercontent.com/assets/3620508/8658026/b27c3b16-29a1-11e5-8e11-609479e75482.JPG)
revision: https://github.com/TrinityCore/TrinityCore/commit/88d14034899936f2e0920652bc848dadde502ec0

with old core revision is everything okay 881ebcc
crashlog http://pastebin.com/Py1Lz6HV

**Comment by DDuarte** [comment]
See #15068

**Comment by msoky** [comment]
with old core revision is everything okay https://github.com/TrinityCore/TrinityCore/commit/881ebccdc4a0995732aea2052f85015aebe6f9a2
crashlog http://pastebin.com/Py1Lz6HV

**Comment by msoky** [comment]
@DDuarte https://github.com/TrinityCore/TrinityCore/issues/15068 is absolutely different problem...

**Comment by Aokromes** [comment]
P.S. making screenshots instead posting text makes very hard for people having the same issue searching for the bug.

**Comment by Greymane** [comment]
Happened to me on branch 3.3.5 aswell, problem is on PreparedResultSet and only happens if i try to run world/auth server without compatibility mode on windows 7 (resulting on way higher startup times). Looks like an OS oriented problem.

I managed to hackfix it by commenting these lines on PreparedResultSet, however Shauren said that by doing that i might have some prroblems in the future.

    /*if (m_stmt->bind_result_done)
    {
        delete[] m_stmt->bind->length;
        delete[] m_stmt->bind->is_null;
    }*/

That way i can now load my worldserver on debug mode on less than a minute, where it was 10+ minutes in compatibility mode.

Heres the crashlog:
http://pastebin.com/cKs1Pb4Y

**Comment by DDuarte** [comment]
And why were you running it on compatibility mode?

**Comment by Greymane** [comment]
Issue is now fixed, the  problem was CMAKE grabbing the wrong MySQL libs and headers:

> Found MySQL library: C:/Program Files/MySQL/MySQL Connector C 6.1.5/lib/libmysql.lib
> Found MySQL headers: C:/Program Files/MySQL/MySQL Connector C 6.1.5/include

Thanks to aokromes for the walktrough :+1:

**Comment by tkrokli** [comment]
Thanks for pointing that out. I noticed the same, CMake 3.2.3 always picks
C:/Program Files/MySQL/MySQL Connector.C 6.1/
over C:/Program Files/MySQL/MySQL Server 5.6/
... on recent source, when configuring the source to a new and empty build folder.
I just compared with an old source, 147a959f51ca68d28d7ad298af9982ccc8d90101 from March 9., it did not happen there.
I always correct this manually when I configure and generate a build to an empty folder.
